### ISSUE TITLE
linCmt() issues

### ISSUE THREAD

**Issue by mattfidler** [nlmixr2/nonmem2rx#161]
All 3 fail:

```r
modN001  <- nonmem2rx("runN001.ctl",lst=".res", save=FALSE)
modN001b <- nonmem2rx("runN001b.ctl",lst=".res", save=FALSE)
modN001c <- nonmem2rx("runN001c.ctl",lst=".res", save=FALSE)
```

- The first one doesn’t seem to like $SUBS ADVAN1,TRANS2
- If I recode without TRANS2 (runN001b.ctl) it complains that it is missing a variable w
- If I recode with the additive error fixed to zero (runN001c.ctl) it claims that a variable ‘eps1’ should be available and then that object ‘time’ cannot be found

Data file is `Bolus_1CPT` from the comparison paper

@RikSchoemaker 
[runN001x.zip](https://github.com/nlmixr2/nonmem2rx/files/12420697/runN001x.zip)

**Comment by mattfidler** [comment]
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/fidlema3/AppData/Local/Temp/1/msohtmlclip1/01/clip_filelist.xml">
<link rel=themeData
href="file:///C:/Users/fidlema3/AppData/Local/Temp/1/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/fidlema3/AppData/Local/Temp/1/msohtmlclip1/01/clip_colorschememapping.xml">





</head>

<body lang=EN-US style='tab-interval:.5in;word-wrap:break-word'>




R version: | R version 4.3.1   (2023-06-16 ucrt)
-- | --
Platform: | x86_64-w64-mingw32
Operating system: | Windows 11 x64 (build   22000)
Attached R packages | Version
babelmixr2 | 0.1.1
base | 4.3.1
cowplot | 1.1.1
data.table | 1.14.8
datasets | 4.3.1
ellipse | 0.5.0
graphics | 4.3.1
grDevices | 4.3.1
grid | 4.3.1
hexbin | 1.28.3
lattice | 0.21-8
latticeExtra | 0.6-30
methods | 4.3.1
nlmixr2 | 2.0.9
nlmixr2data | 2.0.7
nonmem2rx | 0.1.2
RDCOMClient | 0.96-1
stats | 4.3.1
utils | 4.3.1




</body>

</html>

**Comment by mattfidler** [comment]
Hi @RikSchoemaker, 

With the development version of `nonmem2rx`, and the fix #164 I can now import the first model.

The second and third model don't complain for me either, so I'm assuming this was fixed somehow in the `nonmem2rx` unless you are trying to convert to `nlmixr2`.  If that is the case I need a more complete reproducible example
